Batch File/Batch File.sublime-syntax

```diff
@@ -20,12 +21,21 @@ contexts:
       scope: keyword.control.conditional.dosbatch
     - match: \b(?i)for\b
       scope: keyword.control.repeat.dosbatch
-    - match: \b(?:EQU|NEQ|LSS|LEQ|GTR|GEQ)\b
-      scope: keyword.operator.dosbatch
-    - match: "[|]"
+
+  operators:
+    - match: '%(~(?:[fdpnxsatz]*|\$PATH:*)*)?(\d)|%\*'
+      scope: keyword.operator.expansion.dosbatch
+      captures:
+        1: constant.language.modifier.dosbatch
+        2: constant.numeric.dosbatch
+    - match: '@(?=\S)'
+      scope: keyword.operator.at.dosbatch
+    - match: \b(?i:EQU|NEQ|LSS|LEQ|GTR|GEQ|==)\b
+      scope: keyword.operator.comparison.dosbatch
+    - match: '[|]'
       scope: keyword.operator.pipe.dosbatch
-    - match: '&>|\d*>&\d*|\d*(>>|>|<)|\d*<&|\d*<>'
-      scope: keyword.operator.redirect.shell
+    - match: '<&?|>[&>]?'
+      scope: keyword.operator.redirection.dosbatch
```

Comment:
I removed the handle digits as I didn't feel they belonged as part of the operator